docs/building.md

```diff
@@ -0,0 +1,78 @@
+# Overview
+The following document contains information for building *rdap-ingressd* from
+both source and as a Docker container.
+
+# Requirements
+The following requirements need to be available in order to build the project.
+
+- [Git](https://git-scm.com/)
+
+From Source:
+
+- Java 8 or higher. Supports both Oracle JDK and OpenJDK
+- [Maven](https://maven.apache.org/) 3.5 or higher
+
+Docker Container:
+
+- [Docker CE](https://www.docker.com/community-edition) 17 or higher
+
+# Obtaining The Source Code
+
+The first step in building *rdap-ingressd* is to obtain the source code with
+git.
+
+```
+git clone https://github.com/APNIC-net/rdap-ingressd
+```
+
+# Building & Running From Source
+
+## Building
+*rdap-ingressd* is built using maven. To create a new build of the project
+please run the following maven command.
+
+```
+mvn package
+```
+
+The projects jars have now been created and can be executed.
+
+## Running
+The project can be executed in one of two ways. The first is through maven using
+spring-boot or by executing the create jar directly with java.
+
+Executing with maven:
+
+```
+mvn spring-boot:run
+```
```

Comment:
I think documenting a curl command that demonstrates the service would be useful.